client/views/windows/help.tpl

```diff
@@ -4,6 +4,26 @@
 <div class="container">
 	<h1 class="title">Help</h1>
 
+	<h2>About The Lounge</h2>
+
+	<p class="about">
+		{{#if gitCommit}}
+			<p>The Lounge is running from source (commit <a href="https://github.com/thelounge/lounge/tree/{{gitCommit}}" target="_blank" rel="noopener"><code>{{gitCommit}}</code></a>) based on <strong>{{version}}</strong></p>
+			<p><a href="https://github.com/thelounge/lounge/compare/{{gitCommit}}...master" target="_blank" rel="noopener">Compare changes between <code>{{gitCommit}}</code> and <code>master</code> to see what you are missing</a></p>
+			<p><a href="https://github.com/thelounge/lounge/compare/{{version}}...{{gitCommit}}" target="_blank" rel="noopener">Compare changes between <code>{{version}}</code> and <code>{{gitCommit}}</code> to see the changes made</a></p>
+		{{else}}
+			<p>The Lounge is running <a href="https://github.com/thelounge/lounge/releases/tag/{{version}}" target="_blank" rel="noopener"><strong>{{version}}</strong></a></p>
```

Comment:
I think I prefer the current:

<img width="386" alt="screen shot 2017-12-14 at 02 23 24" src="https://user-images.githubusercontent.com/113730/33980224-e139fe74-e075-11e7-8c6c-96036b5ea579.png">

over:

<img width="245" alt="screen shot 2017-12-14 at 02 23 33" src="https://user-images.githubusercontent.com/113730/33980229-e7474452-e075-11e7-85a5-f54857496d17.png">

The call-to-action link seems more explicit than the link on the version.

Reply:
Agreed re the placement of the link to release notes, but not re the wording of the version string. "The Lounge is in version" sounds really, really strange, and I think it would be better phrased "The Lounge is at version" or "The Lounge v2.x.x".

Reply:
My English is really, really strange, can't argue with that 😁 

"Version 2.x.x" works too. Could also do "Version 2.x.x (stable)" or "Version 2.x.x-pre.n (pre-release)" but that adds a step.

There are alternatives, like putting the version number in the title line (with a `<small>` or aligned to the right or something) and then the release notes link can be with the list of other links.
I don't have a strong opinion and don't want to bikeshed, so :man_shrugging:

Reply:
I tweaked it a bit:

![14-123336357](https://user-images.githubusercontent.com/613331/33992570-d4fecafc-e0db-11e7-8f41-d0826d4ffef4.png)

Reply:
lol you left as-is what we commented about and changed what we didn't comment about :D 

I would definitely leave the links as is instead of button styles.
Common UX (I know I'm gonna get accused to pull UX rules out of my 🍑 again lol, but there are a bunch of good reads on the subject) recommends to use buttons for actions, and links for routing/navigating. All 3 are navigations here (all outside the app), even the last one, which doesn't do the action of reporting a bug, but send the user to a place where to report a bug.
Of course, it's UX / design so it's not black & white, we'll always find counter-examples, but I think in this case it's particularly relevant.

The first button is fine as a button because of the "Check for updates" action.

Reply:
Earlier you were showing to make "See release notes" a link after the version.

So I don't know what you want anymore.

Reply:
> Earlier you were showing to make "See release notes" a link after the version.
> So I don't know what you want anymore.

I'm not sure where the contradiction is. "See release notes" makes sense as a link, but since you're bundling it with checking for updates as "View release notes & check for updates", it's fine as a button.

I see you have made more tweaks, let me take a look.